deployment/README.md

```diff
@@ -0,0 +1,22 @@
+# Container Execution
+
+The commands in this document assume you are in the root directory of this
+ repository.
+
+### Image building
+
+
+```shell
+$> docker build -t zambeze/zambeze -f deployment/Dockerfile .
```

Comment:
Question, do you think it is best to place this in a deployment folder or a different folder name i.e. like development, compose or something else, this is not really a problem, just brainstorming. Deployment to me makes me think of production.

Reply:
Again it's been a convention within my former colleagues to always place Dockerfiles, compose files, k8s files, etc etc under a folder called `deployment`. I'm not sure whether there's a different convention and we can definitely change/create our own.